components/agenda/src/Agenda.svelte

```diff
@@ -1426,7 +1427,7 @@
                       </span>
                     {/if}
                     <p>
-                      {@html event.description || ""}
+                      {@html DOMPurify.sanitize(event?.description ?? "")}
```

Comment:
Is this optional chaining required for `event`? This element is inside of an `{#each allDayEvents as event}`

Reply:
oh yeah, good catch. It isn't needed.